app/controllers/api/v1/concerns/authenticator.rb

```diff
@@ -50,15 +50,19 @@ def current_api_user
                 Rails.logger.error "Error retrieving user api credentials: #{e.class.name}: #{e.message}"
               end
             end
-            # check for token expiry and unset user if expired
-            if user.api_access_token_expired?
+            # check for token expiry and unset user if expired/timed out
+            if user.api_access_token_expired? || user.api_access_token_timed_out?
```

Comment:
The difference between "expired" and "timed out" is unclear without looking up the docstring for `api_access_token_timed_out`.  

Naming the method `api_access_token_idle_timed_out` (or something to that effect) would make these easier to distinguish at a glance.